tests/kilonova_2d_2dgrid_xcomgammaphotoion_inputfiles/results_md5_job0.txt

```diff
@@ -1,22 +1,38 @@
-77b24c09dc260ab6a15f6b6bd96b5124  absorption.out
-897316929176464ebc9ad085f31e7284  bflist.out
-6e1ecb4e4c0127a59a8459b0b5b5623c  deposition.out
-4260ede324f222b0797fa9591057d223  emission.out
-0d8616ac0e315cf600d367dac8f8dc21  emissiontrue.out
-19b974363d361a1a847957b21f13ac25  gamma_light_curve.out
-b1ffa6ae3229fee1ce5c247f0129b7b0  gammalinelist.out
-29ac1cb06f3139df7cbca0bbdb426e1c  grid.out
-084494bc151a07fcbb55fee5e51a70a0  light_curve.out
-3d4b61fa5c0c8987526ebd71787cb8ea  linestat.out
-d80043ce6bedca56511b9fd3f7c0a49f  modelgridrankassignments.out
-9eb60a85900f25e261f2f6c21ea97029  nuclides.out
-302b39d9eb652f8ef1897ea35e0cf393  packets00_0000.out
-ef82d510e191f917a9fccbaea973bee6  packets00_0001.out
-719673d427ff38b5108627146f03b007  packets00_0002.out
-01ad0cdfdc7fa2a1b582d1950e196e2a  packets00_0003.out
-b064404636a58abbe7ef8c2d3c8f8861  spec.out
-a351f1711fecd60c023d0ba7332092db  timesteps.out
-0117aeb9c41c88bd194f10d97723a233  job0/estimators_0000.out
-15e15608abcfea9e502aad4e49e64f68  job0/estimators_0001.out
-d61df09530ecefa7403c5098ba9ad263  job0/estimators_0002.out
-779d1553ba57f3893e24af0e6a62bb38  job0/estimators_0003.out
+47084cf180354cde87acd77d65a069b4  absorption.out
+beb8ea349c4eca205b909cdf1d3b3fce  absorptionpol.out
+c0604236aa7df99cda43e5c7bbe7b6f4  bflist.out
+d67c16d80b0823df73360efca6d7df0d  deposition.out
+ec9d78a73b34d454310a8c732d67c463  emission.out
+e50750ff78ca2a0a51bd5920b2c97b43  emissionpol.out
+a50dda3bbcb443690771b005db512534  emissiontrue.out
+79810a73ea0ec395583d0447778da278  gamma_light_curve.out
+6cc9b78b6c2f9416c88ae50f3aa0a89d  gamma_spec.out
+20dadbc26e8f98511b3a587798c9e190  gammalinelist.out
+20bddb22b6f084a7abc105c210366bfd  grid.out
+ccec0d3d09a535709f79a55626983f07  light_curve.out
+87aa0a1a8eebc5b15f95c5c7ba6325a3  linestat.out
+41a1c375110d8edbf421edca3fc426e8  modelgridrankassignments.out
+bbcd1415c9d43c008cff65da19b32bab  nuclides.out
+e41f1fc941c356a6c5bf5181da3f198b  packets00_0000.out
+57544986a55bf4417a7fbbfa621277a6  packets00_0001.out
+fb633ddbc79e1575aa18c29caed5fee0  packets00_0002.out
+7a3fd051f7ab40c65d698cd935b054d7  packets00_0003.out
+924b321ca64ae409e3b2f85853f8478e  spec.out
+146269dfdbd9e38c096a4f47f9b15890  specpol.out
+bc01f046eab9bf3802149e0ff63d5069  timesteps.out
+a8b1711422741ef5662ce0993825d396  vpackets_0000.out
```

Comment:
Where did you get these checksum files? Did they definitely get generated on the GH CI? virtual packets are distabled in all of the kilonova tests.